drivers/gpu/drm/panel/panel-ilitek-ili9881c.c

```diff
@@ -728,10 +936,10 @@ static int ili9881c_prepare(struct drm_panel *panel)
 	msleep(5);
 
 	/* And reset it */
-	gpiod_set_value(ctx->reset, 1);
+	gpiod_set_value_cansleep(ctx->reset, 1);
```

Comment:
Switching to gpiod_set_value_cansleep is not directly part of adding support for your panel, and therefore should be a separate patch.